adapters/persistence/adapter-maria/src/main/java/domain/Member.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
```

Comment:
```suggestion
@Table("member")
```
Table 애노테이션이 추가가 필요하다고 생각됩니다.

그리고 DB 테이블 네이밍 컨벤션은 스네이크 케이스를 제안 드려봅니다.

Reply:
@Table annotation 이 없으면 기본 entity 명으로 테이블이 생성되는걸로 알고 있는데,
명시해 주는게 낫다고 생각하시나요~?

테이블/컬럼 네이밍 컨벤션은 스테이크로 가시죠~

Reply:
테스트를 해보지 못했는데요.

만약 객체의 네이밍이 `InactiveMember`일 경우 생성되는 테이블의 네이밍이 `inactive_member`으로 만들어진다면 

Entity 애노테이션만 적용해도 좋습니다.

Reply:
네네 ~ 
새로운 PR 에서는 @Table 애노테이션을 명시적으로 붙였습니다.
(명시적으로 붙이는 것이 더 좋다고 생각)

다만 공유차 말씀드리면 스프링 부트에서는 디폴트로 아래와 같이 테이블을 생성합니다!

```
SpringPhysicalNamingStrategy(Default)  
=>  camel case를 underscore 형태로 변경
```

https://mycup.tistory.com/237